### ISSUE TITLE
[Feature] Be able to click maps

### ISSUE THREAD

**Issue by ZeProf2Code** [zDevelopers/ImageOnMap#91]
Hello! It would be great if you add something to send a message/open a link to the player when he clicks a specified map or at a certain point of this map.
Example of a button on a map with particles to indicate that you can click it:
![cba09b2b17f011982ab7d787a4103be9](https://user-images.githubusercontent.com/32982428/83958704-39a55d00-a875-11ea-9f91-cb087b46f5f4.gif)

**Comment by Vlammar** [comment]
Hi, we were planning to use paper as an edit tool to select maps where you want to assign a command (can be a message+link or a teleportation), it will turn the map renderer green for example to show the selection and with a command like /maptool interractat <Command>. It's a good idea to add particules to it, we can use a custom GUI to select the type of particule,
We can also allow to use a more complexe selection (not in game ) in order to have narrower buttons .  

By the way your name seems familiar, are you related to ZeProf2Math?

**Comment by ZeProf2Code** [comment]
@Vlammar No, that was just a joke username but I kept it until now :joy:

**Comment by Vlammar** [comment]
What do you think of a config file like this: 
-Maps: InterractiveMapID
-Button:

> Polygon: (50,60),(40,70),(45,100),(50,240),(150,60)

> Command: tp <user> 500 64 -25
Polygon points are in pixels from the top left corner (x,y), <user> can be used to identify the person who click on the button

**Comment by ZeProf2Code** [comment]
Yeah, that's good.

**Comment by ZeProf2Code** [comment]
it would also be nice to port this addition to the 1.12 release.
